src/job-manager/src/config.ts

```diff
@@ -58,14 +74,23 @@ export function loadConfig(): Config {
     apiEndpoint: env.API_ENDPOINT,
     region: env.AWS_REGION,
     jobTypes: {
-      CRITERIA_POLYGONS: {
-        taskDefinitionArn: env.CRITERIA_POLYGONS_TASK_DEF,
-        clusterArn: env.CRITERIA_POLYGONS_CLUSTER,
-        desiredMinCapacity: env.CRITERIA_POLYGONS_MIN_CAPACITY,
-        desiredMaxCapacity: env.CRITERIA_POLYGONS_MAX_CAPACITY,
-        scaleUpThreshold: env.CRITERIA_POLYGONS_SCALE_THRESHOLD,
-        cooldownSeconds: env.CRITERIA_POLYGONS_COOLDOWN,
-        securityGroup: env.CRITERIA_POLYGONS_SECURITY_GROUP,
+      TEST: {
```

Comment:
Kinda awkward that adding a job type requires so many updates to config schema and type system

Reply:
yeah - it is a bit - as we add more I think I'll naturally refactor things